### PR TITLE
voiceConnection#dispatcher

### PR DESCRIPTION
**Pull request by MrSakamoto-san**

### CODE REVIEW

File: src/client/voice/VoiceConnection.js [hunk 1]
```diff
@@ -117,6 +117,15 @@ class VoiceConnection extends EventEmitter {
   }
 
   /**
+   * The current dispatcher
+   * @type {?StreamDispatcher}
+   * @readonly
+   */
+  get dispatcher() {
+    return this.player.streams.size > 0 ? this.player.streams.last().dispatcher || null : null;
```
Comment (iCrawl) [inline src/client/voice/VoiceConnection.js]: Why specifically the last one? Why not the first? Or the 2nd? Or 3rd?

I don't really know how this is helpful at all, except adding a useless getter.

Comment (MrSakamoto-san) [inline src/client/voice/VoiceConnection.js]: last one as to get latest dispatcher there existing but will change code to be more clear.
Some users hate going all the way to getting dispatcher over voiceConnection so i made shortcut.

**Comment by hydrabolt** [comment]
Sorry, I implemented this myself in the latest commit (a new commit had broken your PR.) Thank you for bringing this to our attention however 😄

**Comment by iCrawl** [comment]
Added in dce8e834.

**Pull request closed without merging**